src/components/header/navbar.module.scss

```diff
@@ -3,6 +3,8 @@
 
 .navbar{
     float: right;
+    margin-bottom: 0;
+    background-color: rgb(34, 34, 34);
 }
 
 .navbutton{
```

Comment:
try to avoid float. Use flexbox instead (for future)

Reply:
float is replaced with flex, as you recommended